### PR TITLE
[WIP] Re-adding fabric8-checks for health/readiness

### PR DESCRIPTION
**Pull request by aperullo**
**Work in Progress**

#### What does this PR do?
This Pull Request is part of an investigation into enabling the fabric8 health and readiness checks. 
It is a work-in-progress and this PR is to allow running through the CI.

#### Who is reviewing it? 
@rfding 
@pvargas 

#### Ask 2 committers to review/merge the PR and tag them here.
@oconnormi
TBD

#### How should this be tested?
TBD

#### Any background context you want to provide?
This feature adds endpoints that can be used to determine the readiness and health of ddf, such as if it is currently installing features, or if any bundles failed to start.

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by aperullo** [comment]
build now

File: features/apps/src/main/feature/feature.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
         <feature>ddf-branding</feature>
         <feature>platform-app</feature>
         <feature>admin-app</feature>
+		<feature>fabric8-karaf-checks</feature>
```
Comment (vinamartin) [inline features/apps/src/main/feature/feature.xml]: ❓ Formatting?

Comment (aperullo) [reply features/apps/src/main/feature/feature.xml]: Odd, that looked right in my text editor. Fixed, thanks Vina!

**Comment by aperullo** [comment]
build now

**Comment by oconnormi** [comment]
Had to do some searching in the logs, initially i couldn't find anything wrong but it looks like there was a single failure in the itests.

```
Failed tests: 
  TestFederation.testSingleUserDownloadSameProductSyncAndAsync:1961->verifyCswStubCall:2028 Expected to happen 1 time(s), but happened 2 times instead```

**Comment by aperullo** [comment]
Rebasing. Build now

**Pull request closed without merging**